src/templates/aids/detail.html

```diff
@@ -184,6 +185,31 @@ <h3>{{ _('Data origin') }}</h3>
                     </div>
                 </div>
             {% endif %}
+
+            {% if programs or financers %}
+            <div id="logos">
+                {% if programs %}
```

Comment:
Sauf erreur, cette ligne n'est pas utile. Si la liste `programs` est vide, la boucle `for` ne fera rien. Cela dit, c'est vraiment pour trouver un truc à dire.